### PR TITLE
add issued-credentials terraform with permission to db

### PR DESCRIPTION
**Pull request by SunilDGDS**
### What changed

Update lambda to add new issued-credentials.tf , with permissions to access issuedcretential table

- [PYI-560](https://govukverify.atlassian.net/browse/PYI-560)

### CODE REVIEW

File: terraform/lambda/issued-credentials.tf [hunk 1]
```diff
@@ -0,0 +1,20 @@
+module "issued-credentials" {
+  source      = "../modules/endpoint"
+  environment = var.environment
+
+  rest_api_id            = aws_api_gateway_rest_api.ipv_internal.id
+  rest_api_execution_arn = aws_api_gateway_rest_api.ipv_internal.execution_arn
+  root_resource_id       = aws_api_gateway_rest_api.ipv_internal.root_resource_id
+  http_method            = "GET"
+  path_part              = "request-credentials"
```
Comment (Wynndow) [inline terraform/lambda/issued-credentials.tf]: I'd suggest this be `issued-credentials` to keep the naming consistent.

File: terraform/lambda/issued-credentials.tf [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+  function_name          = "${var.environment}-issued-credentials"
+  role_name              = "${var.environment}-issued-credentials-role"
+
+  allow_access_to_user_issued_credentials_table = true
+  user_issued_credentials_table_policy_arn      = aws_iam_policy.policy-user-issued-credentials-table.arn
+  user_issued_credentials_table_name            = aws_dynamodb_table.user-issued-credentials.name
+
+  credential_issuer_config_parameter_store_key = aws_ssm_parameter.credential-issuers-config.name
+
+}
```
Comment (Wynndow) [inline terraform/lambda/issued-credentials.tf]: Need a newline here

File: terraform/lambda/issued-credentials.tf [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+  path_part              = "request-credentials"
+  handler                = "uk.gov.di.ipv.core.issuedcredentials.IssuedCredentialsHandler::handleRequest"
+  function_name          = "${var.environment}-issued-credentials"
+  role_name              = "${var.environment}-issued-credentials-role"
+
+  allow_access_to_user_issued_credentials_table = true
+  user_issued_credentials_table_policy_arn      = aws_iam_policy.policy-user-issued-credentials-table.arn
+  user_issued_credentials_table_name            = aws_dynamodb_table.user-issued-credentials.name
+
+  credential_issuer_config_parameter_store_key = aws_ssm_parameter.credential-issuers-config.name
```
Comment (Wynndow) [inline terraform/lambda/issued-credentials.tf]: Don't need this parameter for this lambda.

File: terraform/lambda/issued-credentials.tf (hunk 1 above)
Comment (SunilDGDS) [reply terraform/lambda/issued-credentials.tf]: renamed path to issued-credentials

**Review by Wynndow — APPROVED** [review]

**Pull request merged**